pkg/monitor/intervalcreation/pod.go

```diff
@@ -142,6 +146,26 @@ func (t podLifecycleTimeBounder) getEndTime(inLocator string) time.Time {
 	return t.delegate.getEndTime(locator)
 }
 
+func (t podLifecycleTimeBounder) getPodCreationTime(inLocator string) *time.Time {
+	podCoordinates := monitorapi.PodFrom(inLocator)
+
+	// no hit for deleted, but if it's a RunOnce pod with all terminated containers, the logical "this pod is over"
+	// happens when the last container is terminated.
```

Comment:
I can see how this comment is relevant on lines 172-173 but not sure how it's relevant here.